tests/test_bs.py

```diff
@@ -152,8 +167,13 @@ def setup_bootstrap_in_namespace(namespace, bs_deployment_info, bootstrap_config
     return bs_deployment_info
 
 
-def setup_clients_in_namespace(namespace, bs_deployment_info, client_deployment_info, client_config,
+def setup_clients_in_namespace(namespace, bs_deployment_info, client_deployment_info, client_config, name="client",
                                oracle=None, poet=None, dep_time_out=120):
+    # this function used to be the way to extract the client title
+    # in case we want a different title (client_v2 for example) we can specify it
+    # directly in "name" input
+    def _extract_label():
```

Comment:
this can be a util function, unless there is a special reason to include it whithin this fixture

Reply:
I thought about it, on one of my next commits i'll add a util and convenience files and add a bunch of functions there